src/types/builtin_costs.rs

```diff
@@ -1,4 +1,7 @@
 //! # Builtin costs type
+//!
+//! A ptr to a list of u64, this list will not change at runtime in size and thus we only really need to store the pointer,
+//! it can be allocated on the stack on rust side and passed.
```

Comment:
Does this pointer point to the global declared earlier? If so, why do we need the global?

Reply:
No, the global has a pointer, which points to the list.

This type represents the pointer to the list, to get the BuiltinCosts type for the libfunc that retrieves it (and for one of the withdraws that doesnt have a builtincosts argument) you need to have a global to get the ptr to the list

Reply:
Understood. Thank you.